Makefile

```diff
@@ -206,6 +203,14 @@ install:
 	ln -sf $(POPLOG_VERSION_SYMLINK)/pop/pop/poplog $(EXEC_DIR)/poplog$(VERSION_DIR)
 	# Target "install" completed
 
+.PHONY: install-poplocal
+install-poplocal:
+	mkdir -p $(POPLOCAL_HOME_DIR)
+	( cd poplocal; tar cf - --exclude=.gitkeep . ) | ( cd $(POPLOCAL_HOME_DIR); tar xf - . )
```

Comment:
For my edification, why the `tar` trickery, is `cp` not sufficient? Or is it just to exclude gitkeep?

Reply:
`cp` is rarely the right option. It has a tendency to change things on the copy, such as permissions (use `-p`) and follow symlinks (use `-RP`), not overwrite (use `-f`), and be aliased by well-meaning sysadmins (use `/bin/cp`), fail to modify permissions on overwrite (use `-p` (again)) and use the `umask` (nooo!). And it doesn't help that the options you need to fix things up are not consistent across Unixes. 

`tar` is horrible but it got wired into my brain some 35 years ago and actually does work correctly. And it has all the options you need for this work, like `--exclude`.

So not a fantastic reason. I don't think this is asking for a change & I don't think I need to pay the honesty tax either.